sentry/src/main/java/io/sentry/SentryEvent.java

```diff
@@ -97,6 +97,9 @@
   /** List of breadcrumbs recorded before this event. */
   private List<Breadcrumb> breadcrumbs;
 
+  /** Contexts describing the environment (e.g. device, os or browser). */
+  private final @NotNull Contexts contexts = new Contexts();
```

Comment:
in theory, this is a Nullable field.

Reply:
This has been changed to not-nullable in previous PR.